https://gitlab.com/wireshark/wireshark/-/commit/411df7ca34027d87fe6ecc06a07c816a401c4cb5
https://gitlab.com/wireshark/wireshark/-/issues/17766
https://bugs.gentoo.org/823720

From: Gerald Combs <gerald@wireshark.org>
Date: Mon, 6 Dec 2021 19:01:10 -0800
Subject: [PATCH] CMake: Remove the update_tools_help target.

Remove the update_tools_help target. Despite the comment, the weekly
update job doesn't use it, we don't have targets for our other update
scripts, and it currently causes issues if BUILD_tshark is disabled.
Fixes #17766.
--- a/docbook/CMakeLists.txt
+++ b/docbook/CMakeLists.txt
@@ -33,16 +33,6 @@ set(COMMON_GRAPHICS
 
 set(GUIDE_INSTALL_DIRS)
 
-# This is automatically updated each week. You probably don't need to run it
-# or depend on it.
-add_custom_target(update_tools_help
-	COMMAND ${CMAKE_COMMAND} -E env PYTHONIOENCODING=UTF-8
-		${PYTHON_EXECUTABLE} ${CMAKE_SOURCE_DIR}/tools/update-tools-help.py
-		--program-path $<TARGET_FILE_DIR:tshark>
-	WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-)
-set_docbook_target_properties(update_tools_help)
-
 set(WSUG_TITLE "Wireshark User's Guide")
 
 set(WSUG_FILES
GitLab
